mwafrika/teamwork
# #168401493 feat(view all posts): Display All posts for employees

#### What does this PR do?
Allow employees to view all available post showing the most recent come first. 

#### Description of Task to be completed?
- employees can see all posted articles for all registered user in the system.
- employees can get access to several options that concern them.
- employees can also see connected employee to the system
- employees can view articles of a specific category

#### How should this be manually tested?
- every users have access to see all users and comment on the other's post,
- user cannot edit the post that does not belong to him.

#### Any background context you want to provide?
The system displays the employees who are connected to the internet with green color
The posted articles are grouped by category

#### What are the relevant pivotal tracker stories?
#168401493 
#### Screenshots (if appropriate)
this picture below show post and all employees who are online 
![Screenshot (31)](https://user-images.githubusercontent.com/25848053/64644076-206c9e80-d412-11e9-898e-cc34eb748325.png)

this picture below shows users , posts and the time when the article has been published and also 
shows the posts that has been published before others basing on the time
![Screenshot (32)](https://user-images.githubusercontent.com/25848053/64644158-45f9a800-d412-11e9-86a5-d2c0bbc1a73d.png)

the picture below shows the different categories of the published articles
![Screenshot (33)](https://user-images.githubusercontent.com/25848053/64644209-6a558480-d412-11e9-9613-04d8a7a12719.png)
